src/hashtable.py

```diff
@@ -84,7 +113,10 @@ def resize(self):
 
         Fill this in.
         '''
-        pass
+        new_bucket = [None] * self.capacity * 2
+        for i in range(self.capacity):
+            new_bucket[i] = self.storage[i]
+        self.storage = new_bucket
 
```

Comment:
I don't know if you fixed this already, and didn't push it.
Remember, if you don't change the capacity attribute itself, this method will give you a false positive on the tests